Okta/OktaUtils.swift

```diff
@@ -80,4 +74,9 @@ open class Utils: NSObject {
         }
         return model
     }
+
+    internal class func removeTrailingSlash(_ val: String) -> String {
+        // Removes the URLs trailing slash if it exists
+        return String(val.suffix(1)) == "/" ? String(val.dropLast()) : val
```

Comment:
See this [here](http://swift.sandbox.bluemix.net/#/repl/5a7cd5d5f0e9581a04689cd3)